homework/style.css

```diff
@@ -1,3 +1,139 @@
-.alert-error {
```

Comment:
Why did you delete this class? It still necessary for errors. You use it in line 173 for your JavaScript file.